src/py42/clients/detectionlists/high_risk_employee.py

```diff
@@ -0,0 +1,265 @@
+        resource = u"/highriskemployee/remove"
+        uri = u"{0}{1}".format(self._uri_prefix, resource)
+        return self._session.post(uri, data=json.dumps(data))
+
+    def get(self, user_id):
+        """
+        Get user information.
+
+        Args:
+            user_id (str/int): Id of the user who needs to be added to HRE lens.
+
+        Returns:
+            :class:`py42.sdk.response.Py42Response`
+        """
+        data = {"tenantId": self._user_context.get_current_tenant_id(), "userId": user_id}
+        resource = u"/user/getbyid"
+        uri = u"{0}{1}".format(self._uri_prefix, resource)
+        return self._session.post(uri, data=json.dumps(data))
+
+    def _get_high_risk_employees_page(
+        self,
+        tenant_id,
+        filter_type=None,
+        sort_key=None,
+        sort_direction=None,
+        page_num=None,
+        page_size=None,
+    ):
+        # Overwriting page_size since default value 1000 returns error
+        page_size = 100
+        data = {
+            "tenantId": tenant_id,
+            "filterType": filter_type,
+            "pgNum": page_num,
+            "pgSize": page_size,
+            "srtKey": sort_key,
+            "srtDirection": sort_direction,
+        }
+
+        resource = u"/highriskemployee/search"
+        uri = u"{0}{1}".format(self._uri_prefix, resource)
+        return self._session.post(uri, data=json.dumps(data))
+
+    def get_all(
+        self, filter_type="OPEN", sort_key=None, sort_direction=None,
+    ):
+        """
+        Search High Risk employee list. Filter results by filter_type
+
+        Args:
+            filter_type (str): Valid filter types.
+            sort_key (str): Sort results based by field.
+            sort_direction (str): "ASC" or "DESC"
+
+        Returns:
+            generator: An object that iterates over :class:`py42.sdk.response.Py42Response` objects
+            that each contain a page of users.
+        """
+
+        return get_all_pages(
+            self._get_high_risk_employees_page,
+            "items",
+            tenant_id=self._user_context.get_current_tenant_id(),
+            filter_type=filter_type,
+            sort_key=sort_key,
+            sort_direction=sort_direction,
+        )
+
+
+class DetectionListUserClient(BaseClient):
+    """
+    Administrator utility to manage High Risk employees information.
+    `Support Documentation <https://support.code42.com/Administrator/Cloud/Monitoring_and_managing/Detection_list_management_APIs>`__
+    """
+
+    _api_version = u"v2"
+    _uri_prefix = u"/svc/api/{0}/user".format(_api_version)
+
+    def __init__(self, session, user_context):
+        super(DetectionListUserClient, self).__init__(session)
+        self._user_context = user_context
+
+    def update_notes(self, user_id, notes):
+        """Add or update notes related to the user.
+
+        Args:
+            user_id (str/int): The user_id whose notes need to be updated.
+            notes (str): Note.
```

Comment:
How about instead of `Note.`, we do `User profile notes.`

Reply:
Changed.